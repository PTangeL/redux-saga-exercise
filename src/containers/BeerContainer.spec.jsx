import React from 'react'
import { shallow, mount } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import BeerContainer from './BeerContainer'

describe('<BeerContainer />', () => {
  const props = {
    beer: {}
  }

  const wrapper = shallow(
    <BeerContainer {...props} />
  )

  it('should render with props', () => {
      expect(shallowToJson(wrapper)).toMatchSnapshot()
    })

  it('Method #componentWillReceiveProps() should conditionally call setLoadingBeer(true)', () => {
    //TODO implement test
  })

  it('Method #setLoadingBeer() should set state of loadingBeer', () => {
    //TODO implement test
  })
})
