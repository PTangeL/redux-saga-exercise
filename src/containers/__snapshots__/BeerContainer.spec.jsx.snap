// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BeerContainer /> should render with props 1`] = `
<Beer
  beer={Object {}}
  loadingBeer={true}
  setLoadingBeer={[Function]}
/>
`;
